.github/workflows/regression-report.yml

```diff
@@ -454,10 +454,30 @@ jobs:
 
       - name: Prepare environment
         run: |
-          mv .ci-temp/project.properties ./.ci-temp/contribution/checkstyle-tester/
-          mv .ci-temp/*.xml ./.ci-temp/contribution/checkstyle-tester/
+          mv .ci-temp/project.properties ./.ci-temp/test-configs/checkstyle-tester/
+          mv .ci-temp/*.xml ./.ci-temp/test-configs/checkstyle-tester/
           sudo apt install groovy
 
+      - name: Download DiffTool JAR
+        uses: robinraju/release-downloader@v1.8
+        with:
+          repository: "relentless-pursuit/test-configs"
+          tag: "diff-tool-1.0.0"
+          fileName: "difftool.jar"
+
+      - name: Download patch-diff-report-tool JAR
+        uses: robinraju/release-downloader@v1.8
+        with:
+          repository: "checkstyle/test-configs"
+          tag: "patch-diff-report-tool-0.1-SNAPSHOT"
+          fileName: "patch-diff-report-tool-0.1-SNAPSHOT-jar-with-dependencies.jar"
```

Comment:
This should be downloading from contribution repository, as contribution repo host sources, so we will always release to artifacts of contribution repository

Reply:
release is moved to contribution repo https://github.com/checkstyle/contribution/releases/tag/patch-diff-report-tool-0.1-SNAPSHOT

Reply:
updated